src/client/Microsoft.Identity.Client/Extensibility/AbstractAcquireTokenParameterBuilderExtension.cs

```diff
@@ -30,5 +30,28 @@ public static partial class AbstractConfidentialClientAcquireTokenParameterBuild
 
             return builder;
         }
+
+        /// <summary>
+        /// Binds the token to a key in the cache. L2 cache keys contain the key id.
+        /// No cryptographic operations are performed on the token.
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <param name="keyId">A key id to which the access token is associated. The token will not be retrieved from the cache unless the same key id is presented. Can be null.</param>
+        /// <returns>the builder</returns>
+        public static AbstractAcquireTokenParameterBuilder<T> WithProofOfPossessionKeyId<T>(
```

Comment:
I guess it's fine to fix the typo in the name, as this method wasn't used yet (it's for SAL / IdWeb)

Reply:
Yes, and it's marked as experimental too. Hopefully this is an okay fix.

Reply:
Only small concern is that this adds it to public client flows, where it would not do anything. Probably ok. 

@pmaytak - thoughts on limiting it to CCA?

Reply:
What would you like to do for this? This is blocking working on Id Web which is needed for end of August.